### PR TITLE
GCI (Fixed)

### PR DESCRIPTION
**Pull request by mdj117**

### CODE REVIEW

**Comment by mdj117** [comment]
I had to re write the code and send it to one of my friends to test,
It *should* be working now

File: assets/blocks/RipePineapple.block [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "basedOn": "SimpleFarming:MaturePineapple",
+  "displayName": "Ripe Pineapple",
+  "entity": {
+    "prefab": "RipePineapple"
+  },
+  "tile": "SimpleFarming:PineappleFruit"
```
Comment (oniatus) [inline assets/blocks/RipePineapple.block]: I think the title should be RipePineapple. In this version, the growth stage looks like this:
terasology-161230185258-1152x720

What is the PineappleFruit.png for, if not for the last growth stage? :)

Comment (mdj117) [inline assets/blocks/RipePineapple.block]: The Ripepineapple is the final stage

Comment (mdj117) [inline assets/blocks/RipePineapple.block]: changes made :)

**Comment by Cervator** [comment]
By the way, keep in mind a fruit doesn't *have* to be a bush. The berry bush was just set up as a block (initially a half-block for the immature plant) because, well, it is a bush :-)

You could simply declare the pineapple crop to be a regular billboard style plant where instead of being a block it would be a criss-cross 2d plant texture like other plants you find around the world. That may look more sensible.

**Pull request merged**